R/lru.R

```diff
@@ -61,6 +59,46 @@ LRUcache_class <- R6::R6Class("LRUcache",
       oldest <- names(which.min(times))
       rm(list = oldest, envir = private$data)
     },
+    format_cache = function(){
```

Comment:
please add a space before `{` and reformat the body of the function. The line is too wide and it's confusing